test/index.js

```diff
@@ -1,7 +1,21 @@
 /* global describe, it */
-var assert = require('assert');
+var chai = require('chai');
+var expect = chai.expect;
 var builder = require('../dist/');
```

Comment:
I think that this can just be `require('../')` if the `package.json` has `main` defined as `dist/index.js`.